content/gui/xml/ingame/widgets/captains_log.xml

```diff
@@ -6,9 +6,10 @@
 <HBox name="logbook" max_size="700,508" min_size="700,508" padding="-7" position="168,40">
 	<!-- left page -->
 	<VBox max_size="350,508" min_size="350,508" padding="5" name="logbook_left">
-		<VBox name="custom_widgets_left" max_size="350,472" min_size="350,0"
-			comment="In here, widgets defined in yaml get added on runtime." />
-
+		<ScrollArea name="transparent_scrollarea" min_size="325,468" max_size="325,468">
```

Comment:
No. The Scrollarea contains a VBox of max. 1024 height. It itself should not be that big, but should fit on the site of the logbook above the buttons (and inside the surrounding VBox). You can test it and see what it looks like. From what I expect, it should look like before your fix, because the ScrollArea sees no reason to scroll ;)